.github/workflows/python-unit-tests-coverage.yml

```diff
@@ -0,0 +1,49 @@
+name: Python unit tests coverage
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch'
+        required: true
+        default: 'main'
+  pull_request:
+    paths:
+      - '.github/workflows/python-unit-tests-coverage.yml'
+jobs:
+  build:
+    env:
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+      BRANCH_BASE: ${{ github.base_ref || inputs.base_branch }}
+      PYTHONPATH: /home/runner/work/wazuh/wazuh/api:/home/runner/work/wazuh/wazuh/framework
+    runs-on: ubuntu-latest
```

Comment:
Use `ubuntu-22.04` to avoid a potential issue with `ubuntu-24.04` and the dependencies installation (in master it is ok to keep it in latest becasue the installation is not yet defined)

Reply:
After the changes made in 5705cb63559f969b415102ea8eff605bb287e552 all test passed as expected. Test results: https://github.com/wazuh/wazuh/actions/runs/11480518503/job/31949103196?pr=26470